src/api/java/jni/cvc5_Grammar.cpp

```diff
@@ -0,0 +1,120 @@
+#include "cvc5_Grammar.h"
+
+#include "api/cpp/cvc5.h"
+#include "cvc5JavaApi.h"
+
+using namespace cvc5::api;
+
+/*
+ * Class:     cvc5_Grammar
+ * Method:    deletePointer
+ * Signature: (J)V
+ */
+JNIEXPORT void JNICALL Java_cvc5_Grammar_deletePointer(JNIEnv*,
+                                                       jclass,
+                                                       jlong pointer)
+{
+  delete ((Grammar*)pointer);
```

Comment:
It would be better to use C++-style casts:

```suggestion
  delete static_cast<Grammar*>(pointer);
```

Reply:
Generate this error
```
error: invalid static_cast from type ‘jlong’ {aka ‘long int’} to type ‘cvc5::api::Grammar*’
 delete static_cast<Grammar*>(pointer);
```